packages/modelviewer.dev/src/docs-and-examples/sidebar.ts

```diff
@@ -291,7 +308,7 @@ export function sidebarObserver(docsOrExample: string) {
     let orderIndex = 0;
     document.querySelectorAll('div[id*="docs"]').forEach((section) => {
       const idSplitList = section.getAttribute('id')!.split('-');
-      if (idSplitList.length === 4) {
+      if (idSplitList.length === SIZE_OF_ENTRY_LIST) {
```

Comment:
Maybe `NUMBER_OF_CATEGORIES`? Better yet, Is it possible to deduce this from your json instead of having a parameter?

Reply:
I can actually just assign a specific id that doesn't have other values, instead of this conditional-- fixed